### PR TITLE
use JSX.IntrinsicElements for Checkbox component

### PR DESCRIPTION
**Pull request by pankona**
Refs: https://qiita.com/Yametaro/items/814f40d08e9d30584e20

## Updates

- Use `IntrinsicElements` for shorter code

### CODE REVIEW

**Comment by pankona** [comment]
@kachick 

Please review and merge if there's no concern! 😋

**Comment by kachick** [comment]
The direction sounds good 👍 

But I don't know the detail yet. So I don't approve for now 😋

**Comment by pankona** [comment]
@kachick 

Yes, please review carefully! 💯

File: src/atoms/Checkbox.tsx [hunk 1]
```diff
@@ -1,27 +1,7 @@
...
-      className={className}
-      type="checkbox"
-      name={name}
-      defaultChecked={defaultChecked}
-      onChange={onChange}
-    />
-  );
+const Checkbox: FC<JSX.IntrinsicElements["input"]> = (props) => {
+  return <input type="checkbox" {...props} />;
 };
```
Comment (kachick) [inline src/atoms/Checkbox.tsx:5]: I feel this `Checkbox` component is a needless abstraction.
Directly using `input` is enough and clear to me.

But this change is better than now 👍

**Review by kachick — APPROVED** [review]
LGTM 👍 

We should learn about `JSX.IntrinsicElements` more.

**Comment by pankona** [comment]
Thank you!

**Pull request merged**